packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -47,19 +48,25 @@ const StyledDropdownOverlay = styled(BaseBox)<{
       transform: translateY(${makeSize(props.theme.spacing[3])});
       opacity: 0;
       z-index: 99;
+      min-width: max-content;
```

Comment:
we might have to add `max-width` as well because this way it will never break the sentence into next line

<img width="581" alt="image" src="https://github.com/razorpay/blade/assets/30949385/cfac1809-df53-4577-a8d4-a84a8c096b1e">

Reply:
Ok so the issue was, overlay was taking width of the trigger (that is how it is supposed to be when trigger is SelectInput). However in Button triggers, it should have minWidth and maxWidth. I have created a PR stacked on top of this PR #1232 

@Jaynil1611 can you check if that solves your issue? I will merge that into this PR and we can merge this then.